Source/Flow/Private/Nodes/World/FlowNode_ExecuteComponent.cpp

```diff
@@ -316,24 +314,25 @@ const UActorComponent* UFlowNode_ExecuteComponent::TryGetExpectedComponent() con
 	static_assert(static_cast<int32>(EExecuteComponentSource::Max) == 4, TEXT("Update this code if the enum changes"));
 	switch (ComponentSource)
 	{
-		case EExecuteComponentSource::Undetermined:
-			{
-				return nullptr;
-			}
-		case EExecuteComponentSource::BindToExisting:
-			{
-				return AActor::GetActorClassDefaultComponentByName(ExpectedOwnerClass, ComponentRef.ComponentName);
-			}
-		case EExecuteComponentSource::InjectFromTemplate:
-			{
-				return ComponentTemplate;
-			}
-		case EExecuteComponentSource::InjectFromClass:
-			{
-				return IsValid(ComponentClass) ? ComponentClass->GetDefaultObject<UActorComponent>() : nullptr;
-			}
-		default:
+	case EExecuteComponentSource::Undetermined:
```

Comment:
Reintroduced reverted whitespace

Reply:
Heh, this was done to conform to plugin style which is - I need to recheck that - the same as the engine standard. I won't be touching this now.

In the future, I would re-check the plugin style and adjust everywhere to the same style. I have no personal preference here, other than consistently.